### ISSUE TITLE
Cannot start bird due to syntax error.

### ISSUE THREAD

**Issue by debendraoli** [haproxytechblog/ingress-controller-external-example#2]
Got this strange error while staring bird. Do I need anything else to intsall ?

```
BIRD version: 2.0.8
OS: Arch Linux
```

Systemd logs:
```
Starting BIRD routing daemon...
/etc/bird.conf:18:5 syntax error, unexpected IMPORT
bird: /etc/bird.conf:18:5 syntax error, unexpected IMPORT
bird.service: Control process exited, code=exited, status=1/FAILURE
bird.service: Failed with result 'exit-code'.
Failed to start BIRD routing daemon.
```

**Comment by NickMRamirez** [comment]
I'm not too sure, the `import` keyword has been there since version 1.6. Possibly, there is an invisible character? A Windows-style line ending?

**Comment by debendraoli** [comment]
I don't think it is, probably my bird version I am using likly compiled with ipv6 support.

**Comment by wrkode** [comment]
Having the same problem and there's no DOS line ending.
The syntax in this example refers to Bird version 1.x
Bird 2.x requires a different syntax. I can create a PR to get this addressed. Shall I?

**Comment by NickMRamirez** [comment]
@wrkode Thank you! This repo contains only examples shown on our blog, but if you share your example code, I can update the blog post and documentation.

**Comment by debendraoli** [comment]
I was able to fix this actually.
I have to change the whole bird conf syntax. 

I did implement this on my home server.

I can help if needed.

**Comment by NickMRamirez** [comment]
We have updated the documentation to indicate that we support only the BIRD 1.x syntax.
